data/presets/man_made/wastewater_plant.json

```diff
@@ -19,6 +20,7 @@
     "terms": [
         "sewage*",
         "water treatment plant",
+        "wastewater treatment works",
```

Comment:
Yep, and there are a few other words for this in American English too, “wastewater treatment plant” and “WWTP” being the most common.